cleansweep/core/mailer.py

```diff
@@ -47,9 +46,6 @@ def sendmail(to_address, subject, message, message_html=None, reply_to=None, cc=
     if reply_to:
         headers['Reply-To'] = reply_to
 
-    if message_html:
-        message_html = pynliner.fromString(message_html)
-
```

Comment:
why did you remove this?

This is required for inlining all the CSS that we use in the emails. IIRC, it is required for the emails that we send on various events.